src/Analyzers/Core/CodeFixes/AddAccessibilityModifiers/AbstractAddAccessibilityModifiersCodeFixProvider.cs

```diff
@@ -63,15 +69,22 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
             }
         }
 
-        private class MyCodeAction : CodeAction.DocumentChangeAction
+        private class MyCodeAction : CustomCodeActions.DocumentChangeAction
         {
+#if CODE_STYLE // 'CodeActionPriority' is not a public API, hence not supported in CodeStyle layer.
```

Comment:
hrmm... we should add this to the list.

Reply:
Filed https://github.com/dotnet/roslyn/issues/42431